src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrStateActions = [];
+  const stateClone = { ...state };
+
+  for (const properties of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Якщо в тебе є пакет яблук і ти по черзі дістаєш з них один предмет то цей предмет буде називатись яблуко а не властивість